src/messages.js

```diff
@@ -1,9 +1,11 @@
 export const MESSAGE = {
     browserDisconnected:            'The {userAgent} browser disconnected. This problem may appear when a browser hangs or is closed, or due to network issues.',
+    browsersDisconnected:           'The following browsers disconnected: {userAgents}. This problem may appear when a browser hangs or is closed, or due to network issues.',
```

Comment:
I think it should be something like `cantRunOnDisconnectedBrowsers` and the text should be appropriate, something like: 'Can't run tests on disconnected browsers'. @VasilyStrelyaev will help with message.